tests/integration/cmd_push_test.go

```diff
@@ -298,6 +298,19 @@ var _ = Describe("odo push command tests", func() {
 		})
 	})
 
+	Context("when .gitignore file exists or not", func() {
+		It("should create and push the contents of a named component and include odo-file-index.json path to .gitignore file to exclude the contents", func() {
+			helper.CmdShouldPass("git", "clone", "https://github.com/openshift/nodejs-ex", context+"/nodejs-ex")
```

Comment:
Please use `helper.CopyExample` like in https://github.com/openshift/odo/blob/master/tests/integration/servicecatalog/cmd_link_unlink_test.go#L66.

Reply:
I will add such new case in my test script.

Reply:
@prietyc123 this is just adding unnecessary test case and it is taking longer. Using just `CopyExample` is ok